test.sh

```diff
@@ -188,6 +188,15 @@ if [[ "$RUN" =~ "integration" ]] ; then
   end_context #integration
 fi
 
+if [[ "$RUN" =~ "acme-v2" ]] ; then
+  CERTBOT_REPO=https://github.com/certbot/certbot
+  CERTBOT_DIR=$(mktemp -d -t certbotXXXX)
+  git clone $CERTBOT_REPO $CERTBOT_DIR
```

Comment:
Nit: Why not clone at the specific branch we need instead of having a subsequent `git checkout`. E.g.:
`git clone -b acme-v2-integration $CERTBOT_REPO $CERTBOT_DIR`